.changelog/1265.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:enhancement
+r2_bucket: add support for getting a bucket
+```
+
+```release-note:breaking-change
+r2_bucket: change creation time from string to *time.Time

```

Comment:
why do we need to do this?

Reply:
I believe the standard is formatting time strings as `time.Time` so that users of the SDK are able to work with it easier.